python/pylibcudf/pylibcudf/tests/io/test_csv.py

```diff
@@ -148,7 +148,12 @@ def test_read_csv_dtypes(csv_table_data, source_or_sink, usecols):
     if usecols is not None:
         pa_table = pa_table.select(usecols)
 
-    dtypes, new_fields = _convert_numeric_types_to_floating(pa_table)
+    dtypes, new_fields = _convert_types(
+        pa_table,
+        lambda typ: pa.types.is_unsigned_integer(typ)
+        or pa.types.is_integer(typ),
```

Comment:
*nit*: parenthesise this expression? `(pa.types.is_unsigned_integer(typ) or pa.types.is_integer(typ))`